dcipipeline/pipeline.yml

```diff
@@ -2,8 +2,9 @@
   - name: openshift-vanilla
     type: ocp
     location: agents/openshift-vanilla
+    inventory: agents/openshift-vanilla
     outputs:
-      hosts: agents/openshift-vanilla/outputs/localhost/etc/hosts
```

Comment:
we should not change it here in the example pipeline because it will make it fail.

Reply:
@fredericlepied should we create an example-pipeline.yml and a real life pipeline file pipeline.yml where Bill will add the dci-openshift-agent stuffs ?

Reply:
I think real life files should be in their own repo.